setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires =
     jupyterlab<4
     jupyterlab_server
     nbclassic<0.4.0
+    PyYAML==6.0
```

Comment:
Rather than an exact version, can this be `PyYAML>=6.0` (or `<=`)?

Reply:
Changed to `PyYAML>=6.0`